lib/views/help/contact.html.erb

```diff
@@ -1,110 +1,97 @@
 <% @title = "Contact us" %>
 
-<%= foi_error_messages_for :contact %>
+<%= render :partial => 'sidebar' %>
 
-<div id="contact_preamble">
+<h1>Contact us</h1>
+<p>Who do you need to contact?</p>
 
-    <% if !flash[:notice] %>
-        <h1>Contact an authority to get official information</h1>
-        <ul>
-            <li><a href="<%= new_request_path %>">Go here</a> to make a request, in public, for information
-            from UK public authorities.</li>
+<div class="contact-page">
+
+    <h2 class="contact-page__goal" id="government">
+        <label class="houdini-label" for="goal1">
+            I want to contact a <strong>government department</strong>
+            – for example, Visas and Immigration, the Home Office, HMRC,
+            or the Department for Work and Pensions.
+        </label>
+    </h2>
 
+    <input class="houdini-input" type="radio" name="goals" id="goal1">
+    <div class="houdini-target contact-page__options">
+        <ul>
+            <li>
+                If you want to make a public Freedom Of Information
+                request to a UK government department, you can
+                <a href="<%= new_request_path %>">start here</a>.
```

Comment:
(ah, that's been copied across from the original and is what's currently on the site - good catch @garethrees)